verifiable/credential.go

```diff
@@ -17,6 +17,7 @@ type Iden3Credential struct {
 	RevNonce          uint64                 `json:"rev_nonce"`
 	CredentialSubject map[string]interface{} `json:"credentialSubject"`
 	CredentialStatus  *CredentialStatus      `json:"credentialStatus,omitempty"`
+	SubjectPosition   string                 `json:"subject_position,omitempty"`
```

Comment:
for AuthBJJCredential SubjectPosition will be 'none' or not included in the claim at all?

Reply:
@vmidyllic I think not included is better.